### DISCUSSION TITLE
How to import LaravelVuePagination library in Vue3 Script Setup

### DISCUSSION THREAD

**Discussion by saugatdhakal** [gilbitron/laravel-vue-pagination#146] [Help & support]
[[((https://stackoverflow.com/q/73351730/14186855))]](https://stackoverflow.com/questions/73351730/how-to-import-laravelvuepagination-library-in-vue3-script-setup)

**Comment by saugatdhakal** [comment]
Is there any solution for it ??

**Comment by bureauvk** [comment]
This should work

Add to app.js:
```
const app = createApp({})
app.component('Pagination', LaravelVuePagination)
app.mount("#app")
```
And in your Vue Component:
```
<Pagination :data="data" @pagination-change-page="getData"></Pagination>`
```
